Formula/r/r2r.rb

```diff
@@ -0,0 +1,167 @@
+class R2r < Formula
+    include Language::Python::Virtualenv
+  
+    desc "Production-ready RAG engine with a sh*t ton of features"
```

Comment:
```suggestion
    desc "Production-ready RAG engine with a shit ton of features"
```
or 
```suggestion
    desc "Production-ready RAG engine lots of features"
```

everyone knows what `shi*t` means

Reply:
True, we kind of chose this description with that in mind though. Will probably keep it to match our main repo.